cookiecutter.json

```
@@ -1,15 +1,29 @@
 {
   "full_name": "First Middle Last",
   "email": "email@example.com",
-  "github_username": "<github username>",
   "autora_contribution_type": ["theorist", "experimentalist", "experiment_runner", "synthetic_data", "custom"],
+  "custom_autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}<custom_autora_contribution_type>{% else -%}N/A - Press Enter to Skip{% endif -%}",
+  "__autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}{{ cookiecutter.custom_autora_contribution_type }}{% else -%}{{ cookiecutter.autora_contribution_type }}{% endif -%}",  
+  "experimentalist_type": [
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}sampler{% else -%}N/A - Press Enter to Skip{% endif -%}", 
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}pooler{% else -%}N/A - Press Enter to Skip{% endif -%}", 
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}custom{% else -%}N/A - Press Enter to Skip{% endif -%}"],
```

If this whole inner list was wrapped in an if/else construct would it simplify the rendered output when it it is NA? 
```suggestion
    "{% if cookiecutter.autora_contribution_type == 'experimentalist' %} ['sampler', 'pooler', 'custom'] {% else -%}'N/A - Press Enter to Skip'{% endif -%}"]
```
Would something like this work?

---

```
"experimentalist_type": "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}['sampler', 'pooler', 'custom']{% else -%}'N/A - Press Enter to Skip'{% endif -%}",
```
This seems to work for me.

---

No, sadly dosen't work. Starting with "{... will always result in a string (instead of a list).

---

Really? It just renders out the list as a string, right? Or does it give you the options to choose? Does it render like this:
1 sampler
2 pooler
3 custom
pick a number[1]:
or render like this:
pick a number[[sampler, pooler, custom]]:

---

Yes, it is pretty annoying, that cookiecutter only has this super basic command line (double annoying since it would be super straight forward to implement these changes), but I sadly think this is as good as we can get it without using cookiesamurai or some other workaround.